Jenkinsfile

```diff
@@ -1,100 +1,53 @@
 #!/usr/bin/env groovy
 
 pipeline {
-  agent { label 'docker&&linux' }
+  agent {
+    kubernetes {
```

Comment:
We  should forget to remove this job and its credentials from [ci.jenkins.io](https://ci.jenkins.io/job/Infra/job/datadog/)